geometry_msgs/Point point
int width
---
bool success
